toolkit/analysis_and_visualization/classification/assets/scripts/app.js

```diff
@@ -0,0 +1,578 @@
+        row.append($("<td>").attr({
+            "class": "blue filter-image " + misclassifiedClass,
+            "data-id": label.id,
+            "data-type": "misclassified"
+        }).text(label.misclassifiedTop1));
+
+        row.append($("<td>").html("<input id=\"id_\"" + id + " name=\"id[" + id + "]\" type=\"checkbox\" value=\"" + id + "\" onClick=\"highlightRow(this);\"></input>"));
+        myTableBody.append(row);
+    });
+
+
+}
+
+// eslint-disable-next-line no-unused-vars
+function filterLabelTable(rowNum, Datavar) {
+    var labelSummaryFiltered = [];
+    var allEmpty = true;
+    // eslint-disable-next-line no-unused-vars
+    $("input[id ^= \"fl_\"]").each(function (element) {
+        var property = this.id.split("_")[1];
+        var value = this.value;
+        var compare = this.getAttribute("data-compare");
+        if (value) {
+            allEmpty = false;
+
+            labelSummary.forEach(function (label) {
+                var compareResult = false;
+
+                if (compare === "contains") {
+                    compareResult = label[property].toLowerCase().includes(value.toLowerCase());
+
+                } else {
+                    compareResult = label[property] == value;
+                }
+                if (compareResult) {
+                    labelSummaryFiltered.push(label);
+                }
+            });
+        }
+    });
+    if (allEmpty) {
+        loadLabelSummary(labelSummary);
+    } else {
+        loadLabelSummary(labelSummaryFiltered);
+    }
+}
+
+// eslint-disable-next-line no-unused-vars
+function clearLabelFilter() {
+    $("input[id ^= \"fl_\"]").val("");
+
+    loadLabelSummary(labelSummary);
+}
+
+// eslint-disable-next-line no-unused-vars
+function highlightRow(element) {
+    var parentRow = element.parentElement.parentElement;
+    parentRow.classList.toggle("highlight-row");
+    console.log(element.parentElement.parentElement);
+
+}
+
+
+//Create copy of imageSummary
+var imageSummaryFiltered = JSON.parse(JSON.stringify(imageSummary));
+
+// eslint-disable-next-line no-unused-vars
+function clearResultFilter() {
+    $("input[id ^= \"fli_\"]").val("");
+    imageSummaryFiltered = JSON.parse(JSON.stringify(imageSummary));
+    loadImageResults(imageSummaryFiltered);
+    updateItems();
+}
+
+function filterResultTable(e) {
+    imageSummaryFiltered = [];
+    var allEmpty = true;
+    if (e) {
+
+        // eslint-disable-next-line no-unused-vars
+        var code = (e.keyCode ? e.keyCode : e.which);
+    }
+    var combinationOp = $("input:radio[name='filterType']:checked").val();
+    var notOp = $("#not-op").is(":checked");
+
+
+    var filters = {};
+
+    // eslint-disable-next-line no-unused-vars
+    $("input[id ^= \"fli_\"]").each(function (element) {
+        var allNames = this.id.split("_");
+        var property_index = null;
+        var prop_name_new = allNames[1];
+        if (allNames[2]) {
+            property_index = parseInt(allNames[2], 10);
```

Comment:
Codacy found an issue: [Identifier 'property_index' is not in camel case.](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6397592)